wiki/Tournaments/RESC/2024/en.md

```diff
@@ -0,0 +1,539 @@
+| Grease Monkey | 1 | **5** | **together we are terrific** | [![#1](/wiki/shared/link/Link.png)](https://osu.ppy.sh/community/matches/114500462) |
+
+### Round of 32
+
+Saturday, 22 june 2024:
+
+| Team 1 |  |  | Team 2 | Match link |
+| --: | :-: | :-: | :-- | :-: |
+| **swaglords** | **5** | 0 | RX 5600 xt | *win by default* |
+| **NRG FiNESSE** | **5** | 1 | 시발Aim | [![#1](/wiki/shared/link/Link.png)](https://osu.ppy.sh/community/matches/114395380) |
+| **Bobik** | **5** | 0 | Mannheimer Among Us | [![#1](/wiki/shared/link/Link.png)](https://osu.ppy.sh/community/matches/114396911) |
+| **kotomiler** | **5** | 0 | cabaré sasoriosu | [![#1](/wiki/shared/link/Link.png)](https://osu.ppy.sh/community/matches/114397695) |
+| **Caramba Chile** | **5** | 3 | rusticks | [![#1](/wiki/shared/link/Link.png)](https://osu.ppy.sh/community/matches/114397669) |
+| **Brzęczyszczykiewicz** | **5** | 2 | 7 fedotoff 1 turk | [![#1](/wiki/shared/link/Link.png)](https://osu.ppy.sh/community/matches/114399253) |
+| **together we are terrific** | **5** | 1 | Germany owc B | [![#1](/wiki/shared/link/Link.png)](https://osu.ppy.sh/community/matches/114399239) |
+
+Sunday, 23 June 2024:
+
+| Team 1 |  |  | Team 2 | Match link |
+| --: | :-: | :-: | :-- | :-: |
+| **ruyunation** | **5** | 0 | kailburn pirates | [![#1](/wiki/shared/link/Link.png)](https://osu.ppy.sh/community/matches/114409782) |
+| **im actually crying right now** | **5** | 2 | fajny bambule | [![#1](/wiki/shared/link/Link.png)](https://osu.ppy.sh/community/matches/114409760) |
+| **ALL-IN IÑAKI** | **5** | 0 | super hamburger | *win by default* |
+| **rich penguins** | **5** | 2 | cute kittens | [![#1](/wiki/shared/link/Link.png)](https://osu.ppy.sh/community/matches/114411190) |
+| **Nipis Madu** | **5** | 1 | DJ Ramon Sucesso | [![#1](/wiki/shared/link/Link.png)](https://osu.ppy.sh/community/matches/114411923) |
+| **fresh off the boat** | **5** | 0 | doge | [![#1](/wiki/shared/link/Link.png)](https://osu.ppy.sh/community/matches/114412691) |
+| **Grease Monkey** | **5** | 1 | Gente Seria: die Fortsetzung | [![#1](/wiki/shared/link/Link.png)](https://osu.ppy.sh/community/matches/114413476) |
+| **Financial Advisors** | **5** | 0 | el hambre y las ganas de comer | [![#1](/wiki/shared/link/Link.png)](https://osu.ppy.sh/community/matches/114414384) |
+| **<< TIDAL WAVE >>** | **5** | 3 | Cowabunga | [![#1](/wiki/shared/link/Link.png)](https://osu.ppy.sh/community/matches/114414350) |
+
+### Qualifiers
+
+Seeding results are calculated by taking the Z-Sums of the teams and placing those linearly between 0 and 1 where 1 is the best performing team and 0 the worst performing team. The results of the Qualifiers can be found in [this spreadsheet](https://docs.google.com/spreadsheets/d/1IevySrIL2gtHzz5jUByiouxqWNpKwwek4dgmK8cIKI8).
+
+## Ruleset
+
+### General Information
+
+1. All matches will be played with ScoreV2 in Head-to-head mode.
```

Comment:
```suggestion
1. All matches will be played with ScoreV2 in head-to-head mode.
```

Reply:
Both other wiki pages, as well as the game itself has this as "Head-to-head" wth the first h capitalised.